joomla/joomla-cms
# 4.1 Child templates 1/2

Pull Request for Issue # .

### Summary of Changes
This PR
- moves Cassiopeia and Atum to the new mode (eg supporting child templates)
- moves the assets for the system templates to the media folder
- ~~patches tinyMCE to use the correct fallback editor.css~~

### Testing Instructions

DO NOT USE PATCHETESTER FOR THIS ONE. Download the package instead 

- If you're using Git:
It needs `npm I` and also a change in the db table `#_template_styles`: the column `inheritable` should become `1` for all the styles of the 2 templates

- If you're installing the installable package
No extra steps

- Check that the Joomla Install doesn't have any visual regression
- Check that the back end template is not missing any assets and doesn't have any visual regression (skip the extension templates, there's a PR https://github.com/joomla/joomla-cms/pull/35998 for that)
- CHANGE THE DEFAULT EDITOR TO NONE!!! (tinyMCE has its own PR https://github.com/joomla/joomla-cms/pull/36011)
- Check that the front end template is not missing any assets and doesn't have any visual regression
- Check the Error pages
- ~~Check that if you delete the file `media/templates/cassiopeia/css/editor.css` and try to edit an article (assuming that tinyMCE is the default editor) the `media/system/css/editor.css` is in the list of loaded CSS files~~

### Actual result BEFORE applying this Pull Request


### Expected result AFTER applying this Pull Request

There are no visual differences, the changes are in the XML definition of the templates and some static files moved to the `media` folder

### Documentation Changes Required

Still needed:
- The com_templates is not supporting yet the new mode PR: https://github.com/joomla/joomla-cms/pull/35998
- There is another PR for tinyMCE and the editor.css https://github.com/joomla/joomla-cms/pull/36000
